package.json

```diff
@@ -83,7 +85,7 @@
         "env",
         {
           "targets": {
-            "node": 6.4
+            "node": "6.4"
```

Comment:
I remember thinking so to, but see the errors  this gives:

Reply:
I know that travis is dealing so. But I really dont get why. This is more a travis problem then a bug. Because as mentioned in the documenation it is recommended to use a string. 
What do you want to do now?